popup/component/app.js

```diff
@@ -121,7 +130,7 @@ export default class App extends Component {
     return (
       <div className={ styles.status }>
         <span className={ iconClassName } />
-        <span>{ phrase } { url }</span>
+        <span>{ phrase } local node</span>
```

Comment:
Sorry to complicate this even more, but I think it would also be handy to have a network name displayed there instead, like:

"Connected to Ropsten"